lib/rubocop/cop/rails/enum_hash.rb

```diff
@@ -58,6 +58,17 @@ def enum_name(key)
             key.source
           end
         end
+
+        def source(elem)
+          case elem.type
+          when :str
+            "\"#{elem.value}\""
```

Comment:
I guess `String#dump`  is better than the string interpolation. `String#dump` returns eval-safe string.
Can you replace it?

Reply:
Of course ! Than you for this I tried to find a way to do so !

Reply:
It seems like the Symbol#inspect do exactly the same ! Replaced 👍